ocs_ci/templates/ocs-deployment/subscription.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: ocs-subscription
+  namespace: openshift-storage
+spec:
+  channel: alpha
```

Comment:
Do we need to subscribe through a different channel now that we have passed rc milestone?

Reply:
I didn't change the logic here, it was defined in deploy-with-olm just moved subscription outside of this file but as it was.

If we will need to change the channel later on it will be done in different PR but for now I am not aware of any change of channel with RC build.